plugins/eu.numberfour.n4js.ts.model/model/TypeRefs.xcore

```diff
@@ -578,7 +578,7 @@ abstract class StructuralTypeRef {
 	 * contains the structural members. In this case, properties 'astStructuralType' and 'genStructuralType'
 	 * are empty.
 	 */
-	refers TStructuralType structuralType
+	refers transient TStructuralType structuralType
```

Comment:
I'm not quite sure if this can be transient
@mor-n4 : could you have a look at this?